dev/com.ibm.ws.transport.http/src/com/ibm/wsspi/http/ee7/HttpInboundConnectionExtended.java

```diff
@@ -70,4 +70,20 @@ public interface HttpInboundConnectionExtended extends HttpInboundConnection {
      * @return
      */
     boolean useForwardedHeaders();
+
+    /**
+     * Since Servlet 6.0 : support jakarta.servlet.ServletRequest#getProtocolRequestId()
+     *
+     * @return (int) H2 Stream ID; -1 otherwise
+     *
+     */
+    public int getH2StreamId();
+
```

Comment:
Is this something we expose as SPI? I think we've discussed this previously and the answer is no, I just want to confirm.

Reply:
Also, should this method be more generic? Thinking ahead what about Http 3 in the future? Should this just be getStreamId?

Reply:
changed to getStreamId()